curation-migration-datasync/database/dataService.integration.ts

```diff
@@ -0,0 +1,125 @@
+import { truncateDynamoDb } from '../dynamodb/dynamoUtilities';
+import { dbClient } from '../dynamodb/dynamoDbClient';
+import sinon from 'sinon';
+import * as SecretManager from '../secretManager';
+import { writeClient } from './dbClient';
+import { Knex } from 'knex';
+import { DataService } from './dataService';
+import { CuratedFeedProspectItem } from '../types';
+
+describe('database integration test', function () {
+  //aim of this test is to validate knex integration and assumptions.
+  //use cases will be tested in index.integration.ts
+  let db: Knex;
+  beforeAll(async () => {
+    await truncateDynamoDb(dbClient);
+    sinon.stub(SecretManager, 'getDbCredentials').resolves({
+      readHost: 'localhost',
+      readUsername: 'root',
+      readPassword: '',
+      writeHost: 'localhost',
+      writeUsername: 'root',
+      writePassword: '',
+      port: '3310',
+    });
+    db = await writeClient();
+    await db('curated_feed_topics').truncate();
+    const inputTopicData = [
+      { topic_id: 1, name: 'Business', status: 'live' },
+      { topic_id: 2, name: 'Entertainment', status: 'live' },
+      { topic_id: 3, name: 'Health & Fitness', status: 'live' },
+      { topic_id: 4, name: 'Self Improvement', status: 'live' },
+    ].map((row) => {
+      return {
+        topic_id: row.topic_id,
+        name: row.name,
+        status: row.status,
+      };
```

Comment:
nit: I don't think you should need this map, just can insert directly the above array of objects